shared/ui-components/src/page_components/HeaderBadge.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import HeaderBadge from "./HeaderBadge"
+import { withA11y } from "@storybook/addon-a11y"
```

Comment:
This is built in as of Storybook 6, so the import and decorator should be dropped.